src/lib/support/CHIPSparseArray.h

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
I would remove CHIP from the naming prefix - too much CHIP*.h/cpp within our codebase.